omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-SummaryPage.xml

```diff
@@ -0,0 +1,1732 @@
+            width:200px;
+        }
+    </style>
+
+    <!-- Beging of HTML Form -->
+    <div class="ai-page-frame">
+        <div class="headers" style="text-align: center;">
+            <h3>HIV Care/ART CARD - Summary Page</h3>
+        </div>
+        <div id="tabs">
+
+            <ul>
+                <li>
+                    <a href="#care-entry-section">
+                        Care Entry &amp; Family
+                    </a>
+                </li>
+                <li>
+                    <a href="#art-care-cohort-section">
+                        ART Care Cohort
+                    </a>
+                </li>
+
+                <li>
+                    <a href="#prior-art-section">
+                        Treatment Interruptions
+                    </a>
+                </li>
+
+                <li>
+                    <a href="#drug-allergies-and-art-care">
+                        Drug Allergies &amp; ART Care
+                    </a>
+                </li>
+            </ul>
+
+            <div id="care-entry-section" style="background: #F7F7F7;">
+                <section>
+                    <div class="form-group row">
+                        <div class="form-group col-md-2">
+                            <label><span class="required">*</span>Attending Clinician:</label>
+                        </div>
+                        <div class="form-group col-md-2">
+                        <encounterProvider role="Provider" style="autocomplete"/>
+                            <span style="display: none;">
+                        <encounterLocation default="629d78e9-93e5-43b0-ad8a-48313fd99117"
+                                           order="629d78e9-93e5-43b0-ad8a-48313fd99117"/>
+                        </span>
+                        </div>
+
+                        <includeIf
+                                velocityTest="$fn.globalProperty('ugandaemr.showARTPatientNumberIdentifier') == true">
+                        <div class="form-group col-md-2">
+                                <label>ART Patient No.</label>
+                        </div>
+                        <div class="form-group col-md-2">
+                                <patient field="identifier" identifierTypeId="3c76abfd-5e2d-4194-b0d4-00b73c7fc1ac"
+                                         required="false"/>
+                        </div>
+                        </includeIf>
+                        <div class="form-group col-md-2" required="true">
+                            <label><span class="required">*</span>Pt Clinic #:</label>
+                        </div>
+                        <div class="form-group col-md-2">
+                        <patient field="identifier" identifierTypeId="e1731641-30ab-102d-86b0-7a5022ba4115" required="false"/>
+                        </div>
+                    </div>
+
+                    <div class="row">
+                        <div class="form-group col-md-2">
```

Comment:
@dbaluku as discussed with @solemabrothers do not use col-md-* classes just use col-* classes for compatibility across all device classes 

This should also go into the Technical Guide - Form Development - Bootstrap Styling Guidelines

Reply:
Done